fbw-a32nx/src/wasm/fbw_a320/src/FlyByWireInterface.cpp

```diff
@@ -131,7 +131,7 @@ bool FlyByWireInterface::update(double sampleTime) {
     if (simConnectInterface.wasLastBaroInputRightSide()) {
       if (idFcuEisPanelBaroIsInhg[1]->get()) {
         if (fcuBusOutput.baro_setting_left_inhg.Data != fcuBusOutput.baro_setting_right_inhg.Data) {
-          const DWORD kohlsman = fcuBusOutput.baro_setting_right_inhg.Data * 541.822186666672;
+          const DWORD kohlsman = fcuBusOutput.baro_setting_right_inhg.Data * 541.8224;
```

Comment:
```suggestion
          const DWORD kohlsman = fcuBusOutput.baro_setting_right_inhg.Data * 33.8639 * 16;
```
The compiler will optimise this just fine.